cdap-docs/hydrator-manual/source/creating-pipelines.rst

```diff
@@ -142,8 +141,8 @@ To use Hydrator Studio to create a batch pipeline:
   any of the :ref:`batch source plugins <cask-hydrator-plugins-batch-sources>` or the
```

Comment:
I can't get to the correct line, but starting at line 140, add analytics:
```
your pipeline. In addition to the :ref:`action plugins <cask-hydrator-action-plugins>`,
the :ref:`transform plugins <cask-hydrator-plugins-transformations>`, and certain of
the :ref:`analytics plugins <cask-hydrator-plugins-analytics>`, you can use
any of the :ref:`batch source plugins <cask-hydrator-plugins-batch-sources>` or the
```
You could also change the anchor in the `actions/index.rst` from `cask-hydrator-action-plugins` to `cask-hydrator-plugins-actions` to be consistent. I was sloppy here and did not get them coordinated.